### ISSUE TITLE
`langchain.chains.load_chain` fails with `RuntimeError: Loading from the deprecated github-based Hub ...` in langchain 0.1.14 and langchain-core  0.1.39

### ISSUE THREAD

**Issue by harupy** [langchain-ai/langchain#19947]
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```python
from langchain.chains import LLMChain
from langchain.llms import OpenAI
from langchain.prompts import PromptTemplate

llm = OpenAI()
prompt = PromptTemplate(
    input_variables=["product"],
    template="What is a good name for a company that makes {product}?",
)

chain = LLMChain(llm=llm, prompt=prompt)
chain.save("chain.json")

from langchain.chains.load_chain

chain = load_chain("chain.json")
```

One liner to reproduce this error:

```
docker run --rm python:3.9 bash -c 'pip install langchain && python -c "from langchain.chains import load_chain; load_chain(\"/path/to/chain.json\")"'
```

### Error Message and Stack Trace (if applicable)

```
Traceback (most recent call last):
  File "//a.py", line 16, in <module>
    chain = load_chain("chain.json")
  File "/usr/local/lib/python3.9/site-packages/langchain/chains/loading.py", line 625, in load_chain
    if hub_result := try_load_from_hub(
  File "/usr/local/lib/python3.9/site-packages/langchain_core/utils/loading.py", line 10, in try_load_from_hub
    raise RuntimeError(
RuntimeError: Loading from the deprecated github-based Hub is no longer supported. Please use the new LangChain Hub at https://smith.langchain.com/hub instead.
```

### Description

`langchain.chains.load_chain` fails with `RuntimeError: Loading from the deprecated github-based Hub is no longer supported. Please use the new LangChain Hub at https://smith.langchain.com/hub instead.`.

### System Info

```
# pip list | grep langchain
langchain                0.1.14
langchain-community      0.0.31
langchain-core           0.1.39
langchain-text-splitters 0.0.1
```

**Comment by eyurtsev** [comment]
cc @efriis was the intention to deprecate load_chain entirely or just the hub loading part?

**Comment by efriis** [comment]
Hello and sorry about this! This is fixed here #19964 and will be released as `langchain-core==0.1.40` shortly.
